notebooks/official/experiments/comparing_pipeline_runs.ipynb

```diff
@@ -2,7 +2,8 @@
   "cells": [
```

Comment:
<p>do not hardcode the REGION</p><p> </p><p>REGION = "[your-region]"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>